templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>Bolding every word in every paragraph defeats the purpose of using bold. Please try and copy the style of the other notebooks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
@ngreenwald OK, I can remove the bolding. @HPiyadasa moving forward for the notebooks, it'll be easier to use normal markdown text with bolded words indicating important items, as opposed to different H1/H2 headings.